test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -75,6 +93,25 @@ public void NodeSuggestions_InputPortZeroTouchNode_AreCorrect()
             }
         }
 
+        [Test]
+        public void NodeSuggestions_GeometryNodes_SortedBy_NodeGroup_CreateActionQuery()
+        {
+            var type1 = Model.SearchModel.SearchEntries.Where(x => x.FullName.Contains("Curve.Offset")).FirstOrDefault(); //returns a curve.
+            var node = type1.CreateNode();
+            ViewModel.ExecuteCommand(new DynamoModel.CreateNodeCommand(
+               node, 0, 0, true, false));
+            DispatcherUtil.DoEvents();
+            var nodeView = NodeViewWithGuid(node.GUID.ToString());
+            var searchViewModel = (ViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel);
+            searchViewModel.PortViewModel = nodeView.ViewModel.InPorts.FirstOrDefault();
+
+            var suggestions = searchViewModel.GetMatchingNodes();
+            Assert.AreEqual(SearchElementGroup.Create, suggestions.FirstOrDefault().Group);
+            Assert.AreEqual(SearchElementGroup.Action, suggestions.ElementAt(45).Group);
```

Comment:
So this could break when we add new nodes that output a `Curve`, or its subtype? Are we okay with that? I prefer using FFITarget for this reason but I guess this is fine too.

Reply:
I can try to rewrite - the 45 is just somewhere in the middle - not a specific node.